plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastore.java

```diff
@@ -49,7 +49,7 @@
 
     void updateTableStatistics(HiveIdentity identity, String databaseName, String tableName, Function<PartitionStatistics, PartitionStatistics> update, AcidTransaction transaction);
 
-    void updatePartitionStatistics(HiveIdentity identity, Table table, String partitionName, Function<PartitionStatistics, PartitionStatistics> update);
+    void updatePartitionStatistics(HiveIdentity identity, Table table, String partitionName, AcidTransaction transaction, Function<PartitionStatistics, PartitionStatistics> update);
```

Comment:
Any reason not to use the same parameter order as for `updateTableStatistics` above: `update, transaction` ?

Reply:
As in the previous comment, I changed the order in updateTableStatistics.